pkg/acsengine/engine.go

```diff
@@ -487,12 +487,16 @@ func getParameters(cs *api.ContainerService, isClassicMode bool) (map[string]int
 	return parametersMap, nil
 }
 
+func AddValue(m map[string]interface{}, k string, v interface{}) { addValue(m, k, v) }
```

Comment:
My preference would be to matriculate the existing `addValue` to `AddValue` (and then update all existing usages of `addValue` to `AddValue` accordingly). I understand the "change no existing code" directive, but this kind of change is a minor compromise to that strategy with the most long-term benefit, in my view.